server/api/cart.js

```diff
@@ -0,0 +1,29 @@
+const router = require('express').Router()
+const { models: { Order }} = require('../db')
+module.exports = router
+
+router.get('/:id', async (req, res, next) => {
+    try {
+      const singleOrder = await Order.findOne({
+          where:{
+              userId:req.params.id,
+              isOpen:true
+          }
+      })
+      res.send(singleOrder)
+    } catch (err) {
+      next(err)
+    }
+  })
+
+// router.get('/', async(req, res, next) => {
```

Comment:
Why is this commented out?

Reply:
thats just all the orders for every user, i dont think we need it